Move chunks edits (#769)
timescale/docs: timescaledb/how-to-guides/data-tiering/move-data.md

# Data tiering
PostgreSQL uses tablespaces to determine the physical location of your data. In
most cases, you want to use faster storage to store data that is accessed

> frequently, and slower storage for data this is accessed less often.

Reviewer: ```suggestion
frequently, and slower storage for data that is accessed less often.
```

> In TimescaleDB, you can move chunks between different tablespaces, using the

Reviewer: ```suggestion
In TimescaleDB, you can move chunks between different tablespaces using the
```

[`move_chunk`][api-move-chunk] API call.

<highlight type="note">

> You must be logged in as a super user, such as the `postgres` user, to be able

Reviewer: ```suggestion
You must be logged in as a super user, such as the `postgres` user,
```

to use the `move_chunk()` API call.
</highlight>

## Move data

> To set up data tiering you need to have created the tablespace to use and set
> the storage mount point. You can then use the [`move_chunk`][api-move-chunk] API

Reviewer: ```suggestion
To set up data tiering, you first need to create the new tablespace and set
the storage mount point. You can then use the [`move_chunk`][api-move-chunk] API
```

);
```

> You could also keep the data in `pg_default` but move the index to `history`.

Reviewer: ```suggestion
You can also keep the data in `pg_default` but move the index to `history`.
```

> Alternatively, you could set up a third tablespace called `history_indexes`,

Reviewer: ```suggestion
Alternatively, you can set up a third tablespace called `history_indexes`,
```

> move the data to `history`, and the indexes to `history_indexes`.

Reviewer: ```suggestion
and move the data to `history` and the indexes to `history_indexes`.
```
